gem/lib/earlgrey/configure_earlgrey.rb

```diff
@@ -379,8 +379,8 @@ def copy_swift_files(project, target, swift_version = nil)
       src_swift = File.join(src_root, "Swift-#{swift_version}", src_swift_name)
 
       unless File.exist? src_swift
-        puts_magenta "EarlGrey.swift for version #{swift_version} not found. " +
-            "Falling back to version 3.0."
+        puts_magenta "EarlGrey.swift for version #{swift_version} not found. " \
```

Comment:
That was one of the warnings on here:
https://travis-ci.org/google/EarlGrey/jobs/263713604